src/bindings/python/flux/job/kill.py

```diff
@@ -55,12 +55,12 @@ def cancel_async(flux_handle, jobid, reason=None):
     return Future(RAW.cancel(flux_handle, int(jobid), reason))
 
 
-def cancel(flux_handle, jobid, signum=None):
+def cancel(flux_handle, jobid, reason=None):
```

Comment:
Thanks! I also notice that documentation for the `reason` parameter is missing in the docstring for `cancel` and `cancel_async`. Want to add that to this commit as well?

Reply:
Good catch!  I added docs for that argument, and while I was in there, I went ahead and converted this function to the Google style.   I also added type annotations to all the functions in this file as a new commit.

Reply:
Oh wow, I didn't know about type annotations. I'll have to go read up on that one!